Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py

```diff
@@ -2130,10 +2147,20 @@ def search_detections_command():
             for behavior in demisto.get(detection, 'behaviors'):
                 behaviors.append(behavior_to_entry_context(behavior))
             detection_entry['Behavior'] = behaviors
+
+            if extended_data:
+                detection_entry['Device'] = demisto.get(detection, 'device')
+                detection_entry['BehaviorsProcessed'] = demisto.get(detection, 'behaviors_processed')
+
             entries.append(detection_entry)
+
     hr = tableToMarkdown('Detections Found:', entries, headers=headers, removeNull=True, headerTransform=pascalToSpace)
-    ec = {'CrowdStrike.Detection(val.ID === obj.ID)': entries}
-    return create_entry_object(contents=raw_res, ec=ec, hr=hr)
+
+    return CommandResults(readable_output=hr,
+                          outputs=entries,
+                          outputs_key_field='detection_id',
```

Comment:
I see you changed it to `detection_id` from `ID` - are you sure this is correct?